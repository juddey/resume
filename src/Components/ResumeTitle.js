import React from 'react'
import Typography from '@material-ui/core/Typography'

export default class ResumeTitle extends React.Component {
  render () {
    return <Typography variant='display1'>Justin Lane</Typography>
  }
}
