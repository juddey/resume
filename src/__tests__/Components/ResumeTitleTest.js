import React from 'react'
import {
  createComponentWithIntl,
  shallowWithIntl
} from '../../../src/Support/Utils'
import ResumeTitle from '../../Components/ResumeTitle'

const div = document.createElement('div')
const wrapper = shallowWithIntl(<ResumeTitle />)

test('component exists', () => {
  expect(wrapper.length).toBe(1)
})

test('component structure', () => {
  expect(wrapper.name()).toEqual('WithStyles(Typography)')
  expect(wrapper.children().length).toEqual(1)
})

test('renders to snapshot', () => {
  const tree = createComponentWithIntl(<ResumeTitle />, div).toJSON()
  expect(tree).toMatchSnapshot()
})
