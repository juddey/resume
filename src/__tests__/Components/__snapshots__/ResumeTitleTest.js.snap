// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to snapshot 1`] = `
<h1
  className="MuiTypography-root-1 MuiTypography-display1-5"
  style={
    Object {
      "color": "white",
      "letterSpacing": "2px",
      "textTransform": "uppercase",
    }
  }
>
  Justin Lane
</h1>
`;
